packages/hooks/use-form-item/index.ts

```diff
@@ -0,0 +1,53 @@
+import { inject, computed, getCurrentInstance } from 'vue'
+import { elFormKey, elFormItemKey } from '@element-plus/tokens'
+import { buildProp } from '@element-plus/utils/props'
+import { useGlobalConfig } from '@element-plus/utils/util'
+
+import type { ExtractPropTypes, Ref } from 'vue'
+
+const sizes = ['', 'large', 'medium', 'small', 'mini'] as const
+
+export const useFormItemProps = {
+  size: buildProp({
+    type: String,
+    values: sizes,
+    default: '',
+  } as const),
+  disabled: Boolean,
+} as const
+
+export type IUseFormItemProps = ExtractPropTypes<typeof useFormItemProps>
```

Comment:
```suggestion
export type UseFormItemProps = ExtractPropTypes<typeof useFormItemProps>
```
I think `I` prefix is redundant.

Reply:
This is more like a convention to prefix the type with `I` which stands for `interface` even though there it is a type.

Reply:
But it seems that the type naming convention of this project starts with a capital letter, and no  `I`.

Reply:
Alright, let's update it.

Reply:
Don't be hurry. You only changed one line, and there are other references to `UseFormItemProps`.